ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1141,7 +1141,7 @@
           only_trigger_phrase: true
           trigger_permit_all: true
           status_context: jenkins-multicluster
-          status_url: https://jenkins.antrea-ci.rocks/job/antrea-multicluster-for-pull-request
+          status_url: null
```

Comment:
According to Jenkins document, we can use "--none--" to alert the trigger to use a blank url. Usually it is used in lab jenkins to avoid exposing the job page. But in cloud jenkins we should expose the public job link to developers, so we set it to `null`.